src/pages/index.js

```diff
@@ -116,9 +117,18 @@ export default () => {
           />
         </div>
       </Banner>
-      <div id='about' style={{ minHeight: '100vh' }}>
```

Comment:
Why you removed `minHeight`

Reply:
it just happened accidentally. placed it back!